.github/CODEOWNERS

```diff
@@ -0,0 +1,17 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+# Please add names of code owners here and tag the below
+# Jayesh Nirve
+# Bartek Pacia
+# Isabel Coasta
+* @Techno-Disaster @isabelcosta @bartekpacia
```

Comment:
It's extremely minor suggestion but you could change the order so it matches our usernames :P I hope you get what I mean. 
I didn't even know about such feature and I think it's great, good job.

Reply:
ah sorry I will change that asap. Idk how i missed that.